run

```diff
@@ -1,615 +0,0 @@
-#! /usr/bin/env bash
```

Comment:
I really like removing this file from the repo but there are people still using this script (like Apple M1 users), should we remove it anyway?

Reply:
You are right. Added back.